### ISSUE TITLE
Toggle Buttons do not stack well on small screen

### ISSUE THREAD

**Issue by pebcakerror** [dashframework/dashing#14]
![image](https://cloud.githubusercontent.com/assets/613574/24476630/af68040c-1499-11e7-9513-15c6a0599d77.png)

**Comment by inspiredbytes** [comment]
I did a couple of `@media #{phone} {}` overrides

```
.button-group {
  &.button--smooth {
    .button--radio:first-of-type label {
      @media #{$phone} {
        border-radius: 5px 5px 0 0;
        margin: 0;
      }
    }
    .button--radio:last-of-type label {
      @media #{$phone} {
        border-radius: 0 0 5px 5px;
        margin: 0;
      }
    }
  }

  .button--radio {
    @media #{$phone} {
      display: block;
    }
  }
  &.button-group--border {
    @mixin button-group($color) {
      .button {
        @media #{$phone} {
          display: block;
        }
      }
    }

    &.button-group--green { .button--radio { @include button-group($green); } }
  }
}
```


![image](https://cloud.githubusercontent.com/assets/613574/24505048/1f4bb87a-151e-11e7-9e53-4d4e3f224644.png)

**Comment by jrauh** [comment]
Thanks for this, Jonathan. I’ll add this in our list of feature enhancements for dashing. We should hopefully be able to get this in soon.

Jared Rauh
Senior Product Designer · Team Lead
Samaritan Ministries International
(309) 253-7700

**Comment by jrauh** [comment]
This issue has been resolved with `v1.1.6`.

We have implemented a class called `.button-group--vertical` which will stack your button-group. If you want your buttons to only stack on a mobile phone size, add the class `button-group--vertical_phone` in place of the previously mentioned class to only apply the styles to the phone size.

https://github.com/dashframework/dashing/blob/master/sass/modules/actions/_toggle-button.scss#L96-L137
